Audit/appsettings.Development.json

```diff
@@ -1,9 +1,32 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Debug",
-      "System": "Information",
-      "Microsoft": "Information"
+    "Logging": {
+        "LogLevel": {
+            "Default": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+    "DBconnection": {
+        "url": "localhost",
```

Comment:
This too should be an environment variable

Reply:
Actually, I use Development environment for local use. And Production for docker.

Reply:
In docker we have a "dev" environment and configurations, I think you should use "local" as an environment for your local development